src/test/java/shouty/CoordinateTest.java

```diff
@@ -0,0 +1,14 @@
+package shouty;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+public class CoordinateTest extends TestCase {
+
+    @Test
+    public void testItCalculatesTheDistanceFromAnotherCoordinateAlongXAxis() {
```

Comment:
What I want is a way to run all the tests, like you would with 'mvn test'
On Wed, 16 Mar 2016 at 16:54, Aslak Hellesøy <notifications@github.com>
wrote:

> In src/test/java/shouty/CoordinateTest.java
> <https://github.com/cucumber-ltd/shouty.java/pull/3#discussion_r56371198>:
>
> > @@ -0,0 +1,14 @@
> > +package shouty;
> > +
> > +import junit.framework.TestCase;
> > +import org.junit.Test;
> > +
> > +public class CoordinateTest extends TestCase {
> > +
> > +    @Test
> > +    public void testItCalculatesTheDistanceFromAnotherCoordinateAlongXAxis() {
>
> In IDEA: Right-click method, class file or dir and select run. Make note
> of keyboard shortcut and do it with keyboard next time.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cucumber-ltd/shouty.java/pull/3/files/bb8e5142b1570da73566ee7cba2b8f1911b075b7#r56371198>
>

-- 



The Cucumber logo is the intellectual property of Cucumber Ltd, a limited 
company registered in Scotland, number 456793.

UK Headquarters: Cucumber Ltd, Drumsyniebeg, Lochgoilhead, Cairndow, 
Argyll, PA24 8AN UK.

CONFIDENTIALITY NOTICE: The information in this e-mail is confidential and 
privileged; it is intended for use solely by the individual or entity named 
as the recipient hereof. Disclosure, copying, distribution, or use of the 
contents of this e-mail by persons other than the intended recipient is 
strictly prohibited and may violate applicable laws. If you have received 
this e-mail in error, please delete the original message and notify us by 
email immediately. Thank you. Cucumber Ltd.